app/AccountancyModule/CampModule/templates/Cashbook/default.latte

```diff
@@ -1,28 +1,25 @@
 {block #title}{$event->getDisplayName()} - evidence plateb{/block}
 
 {block #content}
-{include ../ActionMenu.latte}
+{include ../header.latte}
 
-<div class="row">
-    <div class="col-sm-12">
-        <h2>{$event->getDisplayName()}</h2>
-    </div>
-</div>
 {if $missingCategories}
     {control categoryAutocomputedControl}
 {else}
-<div class="row">
-    <div class="col-sm-12">
-        <div class="well well-sm">
-            <a n:if="$isEditable" href="#importHpd" role="button" data-toggle="modal" class="btn btn-primary">
-                <i class="fa fa-user"></i>&nbsp;Načíst příjmy od účastníků
-            </a>
+<div class="row mb-2" n:if="$isEditable">
+    <div class="col">
+        <div class="card">
```

Comment:
Není tady zbytečný to mít v `.row > .col`, když je to na celou šířku?

Možná jen nevím o nějakých edge-casech, ale přijde mi to jako stejná varianta jako ten row a col vypustit.

Reply:
odebrano